src/Worker/APDelivery.php

```diff
@@ -37,6 +37,7 @@ public static function execute($cmd, $target_id, $inbox, $uid)
 		} elseif ($cmd == Delivery::SUGGESTION) {
 			$success = ActivityPub\Transmitter::sendContactSuggestion($uid, $inbox, $target_id);
 		} elseif ($cmd == Delivery::RELOCATION) {
+		} elseif ($cmd == Delivery::POKE) {
```

Comment:
Please move the condition with empty blocks to the final `else` like this:
```php
} elseif (!in_array($cmd, [Delivery::RELOCATION, Delivery::POKE]) {
```

Reply:
This is some kind of reminder to myself to fill these empty lines. And especially the relocation is planned to be realized for AP.